ifttt/helpers.py

```diff
@@ -9,3 +16,19 @@ def is_json(myjson):
             return False
         else:
             return True
+
+    #takes time strings, not objects
+    @staticmethod
```

Comment:
If you want to call a method directly (e.g.: `IfThisThenThatHelpers. calculate_hours_diff()`), then you need to decorate it with `@staticmethod`. 

Normally, you would need to have an instance of the object. e.g.: 

```
helper = IfThisThenThatHelpers()
helper. calculate_hours_diff()
```

Also, notice that static methods don't take `self` as a first argument